torchsde/_brownian/modified.py

```diff
@@ -45,15 +43,39 @@ def levy_area_approximation(self):
         return self.base_brownian.levy_area_approximation
 
 
+# TODO: these checks are very inelegant. Is there a better interface to Brownian motion?
```

Comment:
I think this is alright for now. We can come back at a later time.

Reply:
Yeah, there's definitely more important things. Things should also simplify a lot once the tupled logic gets replaced by cat-tensor logic as per #8.

Reply:
Agree